### ISSUE TITLE
Incorrect uppercasing used for default values of variables used in query

### ISSUE THREAD

**Issue by sadlerjw-bridgit** [apollographql/apollo-ios#3290]
### Summary

When defining a query using a variable with a default value, if the default value is an input type with a field whose name starts with an uppercase letter, the query's generated code uses the wrong case for that letter when initializing the input type.

For example if an `input` is defined with a field `PREFIX__id`, then the generated struct for the input type reformats it to `pREFIX__id` but the query tries to initialize it using `PREFIX__id`.

See reproduction steps for a concrete example.

Ideally, the CLI would detect that an ALL-CAPS name is being used and preserve the first-letter capital, but I realize that would likely be a breaking change (and maybe unreliable), so I'd settle for the generated query using the first-letter-lower-cased name :)

Thanks!

### Version

CLI 1.7.1

## Create files:

`schema/schema.graphqls`:
```graphql
type Query {
    example(
        parameter: Param!
    ) : Response
}

input Param {
    PREFIX__id: String
    PREFIX__name: String
}

type Response {
    something: String
}
```

`queries/query.graphql`:
```graphql
query exampleQuery(
    $parameter: Param = { PREFIX__id : "32" }
) {
    example(parameter: $parameter) {
        something
    }
}
```

`apollo-codegen-config.json`
```json
{
  "schemaName" : "ApolloTest",
  "input" : {
    "operationSearchPaths" : [
      "./queries/*.graphql"
    ],
    "schemaSearchPaths" : [
      "./schema/*.graphqls"
    ]
  },
  "output" : {
    "testMocks" : {
      "none" : {
      }
    },
    "schemaTypes" : {
      "path" : "./generated",
      "moduleType" : {
        "other" : {
        }
      }
    },
    "operations" : {
      "inSchemaModule" : {
      }
    }
  }
}
```

## Then run `apollo-ios-cli generate`, which generates:
`generated/Schema/InputObjects/Param.graphql.swift` (non-relevant bits snipped out)
```swift
public struct Param: InputObject {
  public init(
    pREFIX__id: GraphQLNullable<String> = nil,
    pREFIX__name: GraphQLNullable<String> = nil
  ) {
    __data = InputDict([
      "PREFIX__id": pREFIX__id,
      "PREFIX__name": pREFIX__name
    ])
  }
}
```

`generated/Operations/Queries/ExampleQuery.graphql.swift` (snipped again)
```swift
public class ExampleQuery: GraphQLQuery {
  public init(parameter: GraphQLNullable<Param> = .init(
    Param(PREFIX__id: "32")
  )) {
    self.parameter = parameter
  }
}
```

This causes a Swift compilation error since `Param`'s initializer takes a parameter named `pREFIX__id` but `ExampleQuery` is trying to give it a parameter `PREFIX__id`.

**Comment by calvincestari** [comment]
Thanks for reporting this @sadlerjw-bridgit. We'll probably get this fixed along with https://github.com/apollographql/apollo-ios/issues/3249 so they can go out in the same release.

**Comment by BobaFetters** [comment]
This looks to have been resolved by the work done for #3283 and release in version 1.13.0
